### PR TITLE
Change maintainer from @brian-brazil to @roidelapluie, @mem, @electron0zero

### PR DESCRIPTION
**Pull request by beorn7**
@mem and @electron0zero: please verify that I have used your preferred email address for this purpose. And please approve, if you agree with your nomination. (I cannot mark you as a reviewer yet, but this will change once this is merged and we add you to the org.)

### CODE REVIEW

**Review by roidelapluie — APPROVED** [review]

**Comment by beorn7** [comment]
Sorry, I screwed this one up at first. @matthiasr please ignore the notifications about this one.

And here is the proper text:

Yesterday at the dev-summit, the Prometheus team has proposed @roidelapluie, @mem, and @electron0zero as the new maintainer for this repo after @brian-brazil's retirement from the project.

This change will only be merged after an announcement on the prometheus-developers mailing list and is subject to lazy consensus.

Thank you, @brian-brazil, for the incredible amount of work done in this repository and for the project in general.

**Review by electron0zero — APPROVED** [review]

**Comment by beorn7** [comment]
@mem friendly reminder to review this and, once you are happy, to approve.

**Review by mem — APPROVED** [review]
😃

**Pull request merged**
